source/products/hansemerkur/tchibo/cat-surgery/webapp.rst

```diff
@@ -0,0 +1,22 @@
+Webapp
+======
+
+Example Kasko JS embed code
+---------------------------
+
+.. code:: html
+
+    <script type="text/javascript" src="https://js.kasko.io"></script>
```

Comment:
shouldnt the script be this one and also the same for non tchibo 🙈? `<script type="text/javascript" src="https://eu1.kaskojs.com/v2"></script>`

Reply:
Good catch!